libafl/src/corpus/testcase.rs

```diff
@@ -308,6 +314,16 @@ where
     pub fn set_parent_id_optional(&mut self, parent_id: Option<CorpusId>) {
         self.parent_id = parent_id;
     }
+
+    /// Gets how many objectives were found by mutating this testcase
+    pub fn objectives_found(&self) -> usize {
+        self.objectives_found
+    }
+
+    /// Adds one objectives to the `objectives_found` counter. Mostly called from crash handler or executor.
+    pub fn found_objective(&mut self) {
+        let _ = self.objectives_found.saturating_add(1);
```

Comment:
This doesn't do anything.
https://doc.rust-lang.org/std/primitive.usize.html#method.saturating_add

Reply:
thank you for pointing this out! 👍